celesta-core/src/main/java/ru/curs/celesta/dbutils/Cursor.java

```diff
@@ -212,13 +212,13 @@ public final boolean tryInsert() {
                 loggingManager.log(this, Action.INSERT);
                 for (Column c : meta().getColumns().values())
```

Comment:
Maybe 
```
meta().getColumns().values().stream()
    .filter(c -> c instanceOf IntegerColumn)
    .findFirst()
    .map(c -> (IntegerColumn)c)
    .filter(ic -> ic.getSequence() != null)
    .ifPresent(ic -> _setAutoIncrement(db().getCurrentIdent(conn(), meta())))

Reply:
I don't see any issues with the current version - both versions are equivalent.

Reply:
well, @ioanngolovko why convert everything to streams? I vote to leave it as is. I don't mind for..break cycle.